s29986-PrzemyslawKosiedowski/zajecia1/instalacja_debian_linux.txt

```diff
@@ -0,0 +1,1345 @@
+do szyfrowania danych. 
+
+physical volume for RAID - partycja, która będzie używana jako wolumin w
+macierzy RAID. 
+
+physical volume for LVM - partycja, która będzie używana jako wolumin dla LVM. 
+
+do no use the partition - partycja nie będzie używana ani montowana w systemie
+operacyjnym. 
+
+Strona z porównaniem systemów plików:
+https://en.wikipedia.org/wiki/Comparison_of_file_systems 
+
+MOUNT POINT – punkt montowania dla partycji: 
+
+/ - główny katalog systemowy 
+
+/boot - katalog zawierający niezbędne pliki do rozruchu systemu. 
+
+/home - katalogi domowe użytkownika. 
+
+/tmp - pliki tymczasowe. 
+
+/usr - zawiera większość programów, bibliotek i plików danych związanych z
+użytkownikiem. 
+
+/var - zawiera wszystkie zmieniające się dane, np. logi 
+
+/srv - zawiera dane serwerowe, np. pliki FTP 
+
+/opt - zawiera dodatkowe oprogramowanie, zwykle te niedostarczane przez system
+operacyjny. 
+
+/usr/local - przeznaczony dla ręcznie zainstalowanego oprogramowania
+użytkownika, które nie jest zarządzanie przez menedżera pakietów systemowych. 
+
+Enter manually – pozwala wprowadzić punkt ręcznie, podając ścieżkę dostępu do
+konkretnego katalogu na dysku. 
+
+Do not mount it – pozwala nie montować partycji w systemie; nie będzie
+dostępna do odczytu i zapisu. 
+
+MOUNT OPTIONS – opcje montowania: 
+
+discard – informuje system plików, aby natychmiast usuwał niepotrzebne bloki
+danych z urządzenia w trakcie operacji zapisu; może poprawić wydajność przy
+użyciu dysku SSD. 
+
+noatime - wyłącza aktualizację czasu ostatniego dostępu do plików. Może
+usprawnić działanie systemu. 
+
+nodiratime – robi to samo, ale wyłącznie dla katalogów. 
+
+relatime - aktualizuje czas ostatniego dostępu do plików tylko wtedy, gdy jest
+to konieczne do utrzymania poprawnego działania niektórych aplikacji. 
+
+nodev - uniemożliwia uruchomienie plików urządzeń na partycji, np. dysków czy
+kart sieciowych. 
+
+nosuid – blokuje programom możliwość korzystania ze specjalnych ustawień
+plików, które pozwalają im uzyskać dodatkowe uprawnienia. 
+
+noexec - uniemożliwia uruchomienie plików wykonywalnych na partycji. 
+
+ro – partycja jest montowana w trybie tylko do odczytu (read only).  
+
+sync – wymusza na systemie oczekiwanie, aż dane zostaną fizycznie zapisane na
+dysku przez kontynuacją działania. 
+
+usrquota - pozwala na kontrolę ilości używanej przestrzeni dyskowej przez
+poszczególnych użytkowników poprzez ustawienie limitów. 
+
+grpquota – robi to samo, ale dla grupy. 
+
+user_xattr – pozwala na przypisanie dodatkowych informacji do plików i
+katalogów poza standardowymi metadanymi, np. opis pliku. 
+
+LABEL – ustawia etykietę dla partycji. 
+
+RESERVED BLOCKS – obszar na partycji, który jest zarezerwowany do użytku przez
+system operacyjny.  
+
+TYPICAL USAGE - określa typ partycji: 
+
+standard – standardowa konfiguracja, optymalna dla ogólnego użytku. 
+
+news – konfiguracja zoptymalizowana dla systemów przechowujących pliki
```

Comment:
Chodzi o to, że pliki są małe, szybko-zmienne i krótko-żyjące.
Czyli bardzo szybko dochodzi do fragmentacji danych.
Ale głównym celem tego parametru jest odpowiednie dobranie ilości inodów.
To obiekty w których są przechowywane informacje o zapisanych plikach i katalogach.
Jeśli na dysku jest fizycznie miejsce a nie ma wolnych inodów to i tak nic nowego nie zapiszemy.
Więc dobór rozmiaru/ilości inodów jest bardzo ważny. Ale im więcej inodów będzie tym mniej miejsca będzie to gdzieś te dane muszą zostać zapisane na dysku tak czy inaczej.